prov/gni/src/gnix_cm.c

```diff
@@ -1,6 +1,7 @@
 /*
- * Copyright (c) 2015-2016 Cray Inc.  All rights reserved.
- * Copyright (c) 2015 Los Alamos National Security, LLC. All rights reserved.
+ * Copyright (c) 2015-2017 Cray Inc.  All rights reserved.
+ * Copyright (c) 2015-2017 Los Alamos National Security, LLC.
+ *                         All rights reserved.
```

Comment:
I'd suggest not changing copyright if there are no other text changes.  It helps reduce output of git log and friends too.